code/modules/mob/living/carbon/human/species_types/dwarves.dm

```diff
@@ -0,0 +1,217 @@
+
+GLOBAL_LIST_INIT(dwarf_first, world.file2list("strings/names/dwarf_first.txt")) //Textfiles with first
+GLOBAL_LIST_INIT(dwarf_last, world.file2list("strings/names/dwarf_last.txt")) //textfiles with last
+
+/datum/species/dwarf //not to be confused with the genetic manlets
+	name = "Dwarf"
+	id = "dwarf" //Also called Homo sapiens pumilionis
+	default_color = "FFFFFF"
+	species_traits = list(EYECOLOR,HAIR,FACEHAIR,LIPS,NO_UNDERWEAR)
+	inherent_traits = list() 
+	default_features = list("mcolor" = "FFF", "wings" = "None")
+	limbs_id = "human"
+	use_skintones = 1
+	say_mod = "bellows" //high energy, EXTRA BIOLOGICAL FUEL
+	damage_overlay_type = "human" 
+	skinned_type = /obj/item/stack/sheet/animalhide/human
+	liked_food = ALCOHOL | MEAT | DAIRY //Dwarves like alcohol, meat, and dairy products.
+	disliked_food = JUNKFOOD | FRIED //Dwarves hate foods that have no nutrition other than alcohol.
+	mutant_organs = list(/obj/item/organ/dwarfgland) //Dwarven alcohol gland, literal gland warrior
+	mutantliver = /obj/item/organ/liver/dwarf //Dwarven super liver (Otherwise they r doomed)
+	
+	//Pixel X, and Pixel Y offsets on mob drawing. It handles the offset in update_icons.dm
+	//X is Horizontal, Y is Vertical. These are general offsets, directional offsets do not exist.
+	offset_features = list(
```

Comment:
This is the same as the base species datum, please remove.

Reply:
Removed empty off_set features list